.sass-lint.yml

```diff
@@ -29,7 +29,7 @@ rules:
   no-empty-rulesets: 2
   no-extends: 0
   no-ids: 2
-  no-important: 2
+  no-important: 0
```

Comment:
@jeff-phillips-18  is my one love, my anchor ❗️ excising `!important` is dreadful ☠️ work.